packages/marketplace/package.json

```diff
@@ -25,18 +25,18 @@
     "prepublish": "npm run snyk-protect"
   },
   "dependencies": {
+    "@reapit/connect-session": "1.0.0-alpha.1",
```

Comment:
Think we should remove this, marketplace is already running on v2.0.0 code and has not yet been released